index.bs

```diff
@@ -843,6 +845,12 @@ An attribution source is a [=struct=] with the following items:
 :: Number of [=aggregatable debug reports=] created for this [=attribution source=].
 : <dfn>aggregatable debug reporting config</dfn>
 :: An [=aggregatable debug reporting config=].
+: <dfn>attribution scope limit</dfn>
+:: Null or a positive 32-bit integer representing the number of distinct [=attribution source/attribution scopes=] allowed per [=attribution source/attribution destinations|attribution destination=] for the source's [=attribution source/reporting origin=].
+: <dfn>attribution scopes</dfn>
+:: A [=set=] of [=strings=].
+: <dfn>max event states</dfn>
+:: Null or a positive integer representing the maximum number of [=trigger states=] for [=source type/event=] sources per [=attribution source/attribution destinations|attribution destination=] for the source's [=attribution source/reporting origin=].
```

Comment:
Does this field have to be nullable, or can we provide a suitable default?

Reply:
I think we can default this to 3.

Reply:
Would this algorithm still hold then? https://pr-preview.s3.amazonaws.com/feifeiji89/attribution-reporting-api/pull/1215.html#check-if-source-has-incompatible-attribution-scope-fields

Reply:
Done.

Reply:
I see, we should only compare the fields if attribution_scope_limit is set.

Reply:
The explainer says it’s default to 3, but in the spec the behaviors of explicitly setting to 3 or omitted are different. Is this a little confusing?

Reply:
Please update the algorithm.